meta-mender-core/recipes-mender/mender-configure/mender-configure_1.0.0.bb

```diff
@@ -26,3 +26,14 @@ LIC_FILES_CHKSUM = " \
     file://LICENSE;md5=fbe9cd162201401ffbb442445efecfdc \
 "
 LICENSE = "Apache-2.0"
+
```

Comment:
1) why not move all of these into a common include?
2) is a modification of the `.service` files also required? Hopefully not, just to be sure.

Reply:
> 1. why not move all of these into a common include?

Yes, I thought about it. However I removed all 1.0.x recipes in master instead (see [here](https://github.com/mendersoftware/meta-mender/pull/2138/commits/3778ec511acf1dc8e4182fc467fcc08bd29d7efc)) so it is less of an issue now as it will be copied only in two recipes.

>     2. is a modification of the `.service` files also required? Hopefully not, just to be sure.

I don't think so. The file only contains `/usr/share/mender/...` path.